pkg/test/framework/runtime/components/environment/kube/environment.go

```diff
@@ -210,7 +210,7 @@ func (e *Environment) Start(ctx context.Instance, scope lifecycle.Scope) error {
 		name:             e.s.TestNamespace,
 		annotation:       "istio-test",
 		accessor:         e.Accessor,
-		injectionEnabled: true,
```

Comment:
This doesn't look right. Perhaps we should deploy to a separate namespace? (i.e. suite?)

Reply:
I think we should keep the test namespace injection enabled. This is how we would normally use Istio with.